content/docs/concepts/csidriver/features/powermax.md

```diff
@@ -680,3 +680,90 @@ These are the Container Storage Modules not supported with NVMeTCP protocol:
 - Container Storage Modules Observability
 - Container Storage Modules Application Mobility
 - Metro Replication
+
+## Multiple Avalabilty Zones
+
+Starting with CSM 2.14.0 the PowerMax CSI driver supports multiple availability zones (AZ) for block protocols. NFS is not supported at this time.
+
+This feature supports the use of of a StorageClass that is not associated with any specific PowerMax array or storage resource pool (SRP). Each cluster node must be labelled with topology labels which match the labels in the secret.
+
+Requirements:
+- Only one PowerMax array per availability zone is supported.
+- Every PowerMax array must be labelled with one or more zone labels.
+- Every cluster worker node must be assigned to a zone.
+- The StorageClass does not contain any references to the array ID.
+- The volumeBindingMode must be set to WaitForFirstConsumer.
+
+With this support a single storage class can be used to provision volumes from a pool of PowerMax arrays based on topology information where each AZ has its own PowerMax array. The following represents an example of the secret showing two arrays in different availability zones:
+
+### Secret
+
+```yaml
+storageArrays:
+  - storageArrayId: "000000000001"
+    primaryEndpoint: https://primary-1.unisphe.re:8443
+    backupEndpoint: https://backup-1.unisphe.re:8443
+    labels:
+      topology.kubernetes.io/region: region1
+      topology.kubernetes.io/zone: zone1
+    parameters:
+      SRP: srp_1
+      ServiceLevel: Gold
+  - storageArrayId: "000000000002"
+    primaryEndpoint: https://primary-2.unisphe.re:8443
+    backupEndpoint: https://backup-2.unisphe.re:8443
+    labels:
+      topology.kubernetes.io/region: region1
+      topology.kubernetes.io/zone: zone2
+    parameters:
+      SRP: srp_2
+```
+
+### Labelling Worker Nodes
+
+The worker nodes should have corresponding labels to match an availability zone.
+
+```bash
+# Label each worker node in the cluster
+kubectl label nodes worker-1 topology.kubernetes.io/region=region1
+kubectl label nodes worker-2 topology.kubernetes.io/region=region1
+...
+kubectl label nodes worker-1 topology.kubernetes.io/zone=zone1
+kubectl label nodes worker-2 topology.kubernetes.io/zone=zone2
+```
+
+### Storage Class
+
+For multiple availability zones support the StorageClass does not require details about the PowerMax array.
```

Comment:
This sentence seems to be missing something. Please look at it and see if it needs to be re-written.

Reply:
I reworded it a bit. Please check.